projects/buddy-system.md

```diff
@@ -0,0 +1,98 @@
+
+In general buddy's responsibilities are:
+
+1. Being cool with her/his newcomer :sunglasses:
+2. Being available for her/him
+3. Arrange informal meetings with her/him as often as needed
+4. Follow-up with the buddy system maintainer
+5. Turn to the buddy system maintainer any time if needed
+
+Besides, since the process will span over a period of two months, a more specific list divided by phases will be provided.
+
+### Before newcomers arrival
+The buddy must meet the newcomer’s supervisor in order to get an overview of her/his project and discuss the first tasks:
+
+* Schedule when supervisor will introduce buddy to the newcomer
+* Define tasks on Invenio
+  * One of the following small tasks: read up relevant Invenio documentation, rehearse the presentations, test tutorials, check if any part of the material is outdated (~1 hr)
+  * A medium sized task (~1 day) suitable for pair-programming
+  * A first larger service task in which the supervisor will help
+* Agree on who is going to run the bootcamp following `inveniosoftware/training` repository.
+
+### First day
+
+* Take a coffee, present yourself and your project
+* Explain CERN structure: what is a department, section, group etc...
+* Walk through EDH and common actions
+* Let newcomer ask doubts about anything, the more the better
+* Make newcomer aware of:
+	* Invenio Developer Forum
+	* Friday section coffee
+	* Section lunch
+	* Section events (if there are some coming)
+
+
+### 1st week
+
+* Run the bootcamp with the newcomer (using standard material/presentations) (if agreed with supervisor):
+  * Presentation: Introduction to Invenio, use-cases and digital repositories
+  * Presentation: Communication channels (Gitter, Mattermost etc...) Maintainers, Meetings
+* Guided tutorial: Setting up your development environment
+* Guided tutorial: Get Invenio up and running
+* Guided tutorial: Developing for Invenio, including cookiecutter (+seeing all the files), tiny task, running tests, installing inside Invenio
+* Pair-programming: Solve first small task
+* Tutorial: Submitting a pull request
+* Presentation: Overview of contribution and style guide
+* Pair-programming: Solve medium sized task
+* Presentation: Invenio general architecture overview
+* Presentation: Module overview
+* Plan for pair-programming on a service task
+
+**Remember**, newcomers also need “free” time for setting up their environment, office space etc ...
+
+### 2nd week
+
+* Pair-programming on service task
+
+### 3rd-8th week
+
+* Newcomer should already feel comfortable enough to ask for help whenever needed
+* Be accessible for newcomer
+* Check that pull requests and interaction with the section is appropriate (i.e. according to CERN and Invenio code of conduct)
+* Proactively ask for feedback
+
+### 8th week
+
+* Final feedback meeting with buddy (always take note about how to improve the system)
+* Retrospective meeting about last two months with buddy system maintainer
```

Comment:
I am missing a last point "What happens next?" :) I mean, after the the 2 months the 'official' assistance is over but of course, they can still talk and the newcomer can still ask questions, nope? hehehe :)